### PR TITLE
Fix Help menu action order

### PR DESCRIPTION
**Pull request by mandyedi**
This fixes the issue  **About needs to be the last entry in the Help menu #86**

### CODE REVIEW

File: src/NotepadNext/dialogs/MainWindow.ui [hunk 1]
```diff
@@ -40,7 +40,7 @@
      <x>0</x>
      <y>0</y>
      <width>855</width>
-     <height>22</height>
+     <height>25</height>
```
Comment (dail8859) [inline src/NotepadNext/dialogs/MainWindow.ui:43]: Was this change intentional? Thought I'm not sure this would even have any affect.

File: src/NotepadNext/dialogs/MainWindow.cpp [hunk 2]
```diff
@@ -521,21 +521,22 @@ MainWindow::MainWindow(NotepadNextApplication *app) :
...
+    ui->menuHelp->insertAction(editorInspectorDock->toggleViewAction(), languageInspectorDock->toggleViewAction());
 
-    EditorInspectorDock *editorInspectorDock = new EditorInspectorDock(this);
-    editorInspectorDock->hide();
-    addDockWidget(Qt::RightDockWidgetArea, editorInspectorDock);
-    ui->menuHelp->addAction(editorInspectorDock->toggleViewAction());
+    LuaConsoleDock *luaConsoleDock = new LuaConsoleDock(app->getLuaState(), this);
+    luaConsoleDock->hide();
+    addDockWidget(Qt::BottomDockWidgetArea, luaConsoleDock);
+    ui->menuHelp->insertAction(languageInspectorDock->toggleViewAction(), luaConsoleDock->toggleViewAction());
```
Comment (dail8859) [inline src/NotepadNext/dialogs/MainWindow.cpp:539]: Just a thought, would it be cleaner to use a single call to [insertActions()](https://doc.qt.io/qt-5/qwidget.html#insertActions) something like

```cpp
ui->menuHelp->insertActions(separator, { ___, ___, ___});
```

Haven't used that myself so not sure if there's some technical reason why it isn't viable.

**Review by dail8859 — COMMENTED** [review]
Thanks for the PR! Left a few comments.

File: src/NotepadNext/dialogs/MainWindow.ui (hunk 1 above)
Comment (mandyedi) [reply src/NotepadNext/dialogs/MainWindow.ui:43]: It wasn't intentional. Some of the actions are added by the designer and when I change the order and remove one of the separators (it is put back by code), this height changes.

File: src/NotepadNext/dialogs/MainWindow.cpp (hunk 2 above)
Comment (mandyedi) [reply src/NotepadNext/dialogs/MainWindow.cpp:539]: Do you mean constructing the action object while passing it to insertAction?
InsertActions takes a QAction as a second parameter, but we construct a LuaConsoleDock, which is a QDockWidget. After that hide() needs to be called on it, then we pass to insertAction using the toggleViewAction().
I don't think it could make it simpler or even possible.

Comment (dail8859) [reply src/NotepadNext/dialogs/MainWindow.cpp:539]: That's fine. I'll use the code as is. I'd rather have something working and clean it up later.

**Comment by dail8859** [comment]
Thanks for the PR!
